test/blocks/media/media.test.js

```diff
@@ -67,6 +68,33 @@ describe('media', () => {
       const appStoreCta = medias[5].querySelector('a.app-store');
       expect(appStoreCta).to.exist;
     });
+    it('desktop view has visibile qr code image and no google-play and app-store CTA', async () => {
+      await setViewport({ width: 1200, height: 100 });
+      const qrCodeImg = medias[5].querySelector('img.qr-code-img');
+      const qrCTA = medias[5].querySelectorAll('.qr-button-container');
+      expect(window.getComputedStyle(qrCodeImg).getPropertyValue('display')).not.to.be.equal('none');
+      qrCTA.forEach(async (cta) => {
```

Comment:
Why does this function need to be `async`? `forEach` and `async` can interact in unexpected ways, so I'd remove it if it doesn't serve any purpose. Same for L85 and L94.

Reply:
fixed